src/gui/tray/activitylistmodel.cpp

```diff
@@ -226,49 +226,44 @@ QVariant ActivityListModel::data(const QModelIndex &index, int role) const
                 || a._syncFileItemStatus == SyncFileItem::BlacklistedError) {
-                colorIconPath.append("state-error.svg");
-                return colorIconPath;
+                return colorIconPath.arg("state-error.svg");;
             } else if (a._syncFileItemStatus == SyncFileItem::SoftError
                 || a._syncFileItemStatus == SyncFileItem::Conflict
                 || a._syncFileItemStatus == SyncFileItem::Restoration
                 || a._syncFileItemStatus == SyncFileItem::FileLocked
                 || a._syncFileItemStatus == SyncFileItem::FileNameInvalid
                 || a._syncFileItemStatus == SyncFileItem::FileNameInvalidOnServer
                 || a._syncFileItemStatus == SyncFileItem::FileNameClash) {
-                colorIconPath.append("state-warning.svg");
-                return colorIconPath;
+                return colorIconPath.arg("state-warning.svg");;
             } else if (a._syncFileItemStatus == SyncFileItem::FileIgnored) {
-                colorIconPath.append("state-info.svg");
-                return colorIconPath;
+                return colorIconPath.arg("state-info.svg");
             } else {
                 // File sync successful
                 if (a._fileAction == "file_created") {
-                    return a._previews.empty() ? QStringLiteral("image://svgimage-custom-color/add.svg")
-                                               : QStringLiteral("image://svgimage-custom-color/add-bordered.svg");
+                    return a._previews.empty() ? QStringLiteral("image://svgimage-custom-color/add.svg/")
+                                               : QStringLiteral("image://svgimage-custom-color/add-bordered.svg/");
                 } else if (a._fileAction == "file_deleted") {
-                    return a._previews.empty() ? QStringLiteral("image://svgimage-custom-color/delete.svg")
-                                               : QStringLiteral("image://svgimage-custom-color/delete-bordered.svg");
+                    return a._previews.empty() ? QStringLiteral("image://svgimage-custom-color/delete.svg/")
+                                               : QStringLiteral("image://svgimage-custom-color/delete-bordered.svg/");
                 } else {
-                    return a._previews.empty() ? colorIconPath % QStringLiteral("change.svg")
-                                               : QStringLiteral("image://svgimage-custom-color/change-bordered.svg");
+                    return a._previews.empty() ? colorIconPath.arg(QStringLiteral("change.svg"))
+                                               : QStringLiteral("image://svgimage-custom-color/change-bordered.svg/");
                 }
             }
         } else {
             // We have an activity
             if (a._icon.isEmpty()) {
-                colorIconPath.append("activity.svg");
-                return colorIconPath;
+                return colorIconPath.arg("activity.svg");;
```

Comment:
there is a duplicated `;`
```suggestion
                return colorIconPath.arg("activity.svg");
```